.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,6 @@
+blank_issues_enabled: true
+contact_links:
+  - name: Join our Discord community.
+    url: https://jellysquid.me/discord
```

Comment:
Should be `https://caffeinemc.net/discord`. Please don't link to my personal site. We also need to fix this in the README, but that isn't relevant for this PR.